docs/content/how-tos/rules/FL0075.md

```diff
@@ -0,0 +1,29 @@
+---
+title: FL0075
+category: how-to
+hide_menu: true
+---
+
+# VariableLengthGreaterThanOne (FL0075)
+
+*Introduced in `0.21.1`*
+
+## Cause
+
+Rule to detect usage of 1 letter as variable names, function names, and parameter names.
+
+## Rationale
+
+Variable name represents the information the variable contains. It helps to improve code quality particularly during debugging. Variable length of more than one makes reading code pretty easier.
```

Comment:
it is not that "variable length of more than one makes reading code easier", it's that variable of only one letter give poor info about what the variable contains; it's better expressed the other way around

Reply:
Updated.